addons/account/views/account_move_views.xml

```diff
@@ -693,7 +693,7 @@
                                         <field name="product_uom_category_id" invisible="1"/>
                                         <field name="product_uom_id" string="UoM" groups="uom.group_uom" optional="show"/>
                                         <field name="price_unit" string="Price"/>
-                                        <field name="discount" string="Disc.%" groups="base.group_no_one" optional="show"/>
+                                        <field name="discount" string="Disc.%" optional="hide"/>
```

Comment:
But this will impact both customer invoices and vendor bills, right? The task says customer invoices should not be impacted; has it changed since then?

Reply:
Yes, specs did change.